server/controllers/admin/employees.js

```diff
@@ -0,0 +1,313 @@
+/**
+* The /employees HTTP API endpoint
+*
+* @module categorised/employees
+*
+* @description This controller is responsible for implementing all crud on the
+* employees table through the `/employees` endpoint.
+*
+* @requires lib/db
+*
+* NOTE: This api does not handle the deletion of employees because
+* that subject is not in the actuality.
+*/
+
+'use strict';
+
+var db = require('./../../lib/db');
+
+/**
+* Returns an array of each employee in the database
+*
+* @param {object} request The express request object
+* @param {object} response The express response object
+* @param {object} next The express middleware next object
+*
+* @example
+* // GET /employees : Get list of employees
+* var employees = require('categorised/employees');
+* employees.list(request, response, next);
+*/
+exports.list = function (req, res, next) {
```

Comment:
List methods should return minimal information, unless a `?detailed=1` flag is provided.  This comment will not block this PR because I don't think you wrote this code.   But in the future, this should return minimal.